mirrord/cli/src/config.rs

```diff
@@ -217,3 +221,29 @@ pub(super) enum OperatorCommand {
         namespace: OperatorNamespace,
     },
 }
+
+#[derive(ValueEnum, Debug, Clone)]
+pub enum Format {
```

Comment:
use default here

Reply:
do you mean default_value = Json and instead of Option<Format>, Format?